sdk/src/main/scala/coop/rchain/sdk/casper/Stake.scala

```diff
@@ -0,0 +1,14 @@
+package coop.rchain.sdk.casper
+
+object Stake {
+
+  /**
+    * Check if stake prevails threshold of 2/3
+    */
+  def prevails(stake: Long, totalStake: Long): Boolean = 3 * stake >= 2 * totalStake
```

Comment:
I'm not satisfied with this name but I couldn't find out more appropriate name. Anyway for me such functions is better then using magic constants directly.

Reply:
This is a `supermajority`. I think you are right to use >= here.

While majority is > 50, it is not enough for Byzantine settings https://en.wikipedia.org/wiki/Byzantine_fault. So to be sure that in non trusted environment some decision is made, we have to rely on supermajority, which is > 2/3.

Reply:
Imagine large value for the stake with many validators. In that case Long (int64) values can overflow and calculation will be false. That's the reason why I used floats.

Reply:
> This is a `isSupermajority`. I think you are right to use >= here.

PTAL at [this](https://ru.wikipedia.org/wiki/%D0%97%D0%B0%D0%B4%D0%B0%D1%87%D0%B0_%D0%B2%D0%B8%D0%B7%D0%B0%D0%BD%D1%82%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D1%85_%D0%B3%D0%B5%D0%BD%D0%B5%D1%80%D0%B0%D0%BB%D0%BE%D0%B2#%D0%A0%D0%B5%D0%B7%D1%83%D0%BB%D1%8C%D1%82%D0%B0%D1%82%D1%8B_%D0%B8%D1%81%D1%81%D0%BB%D0%B5%D0%B4%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F_%D0%B7%D0%B0%D0%B4%D0%B0%D1%87%D0%B8). I guess `>` is more correct so I was wrong.